.github/workflows/pr-verify.yml

```diff
@@ -76,26 +76,6 @@ jobs:
       - name: Cargo fmt
         run: make format-check
 
-  #  clippy:
```

Comment:
enabled it but it failed and could not reproduce the error locally. It seems like it might be strictly dependent on rust version being used which might be different from my local and CI.